composer.json

```diff
@@ -24,20 +24,19 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": ">=7.0.8",
         "composer/installers": "^1.2",
         "cweagans/composer-patches": "^1.6.5",
-        "drupal-composer/drupal-scaffold": "^2.5",
         "drupal/console": "^1.0.2",
-        "drupal/core": "^8.7.0",
-        "drush/drush": "^9.0.0",
-        "vlucas/phpdotenv": "^2.4",
+        "drupal/core": "^8.8.0",
```

Comment:
IMHO we could require the exact version, tht however would require to setup automation to update all `drupal/*` dependencies automatically when there's a release so we don't need to manually maintain this on each patch release.

Reply:
I think for general projects the semantic number is not that wrong.